lib/lotus/assets/helpers/image_tag.rb

```diff
@@ -0,0 +1,15 @@
+module Lotus
+  module Assets
+    module Helpers
+      class ImageTag
+        def self.render(configuration, source, html_options = {})
+          definition = configuration.asset(:image)
+          path       = definition.url(configuration, source)
```

Comment:
I am assuming `definition.url` will take care of handling prefix and digest mode urls. Am I right, or is there something that need to be done?